eng/common/scripts/Package-Properties.ps1

```diff
@@ -187,12 +187,17 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
         }
     }
 
+    $existingPackageNames = $packagesWithChanges | ForEach-Object { $_.Name }
```

Comment:
```suggestion
    $existingPackageNames = @($packagesWithChanges | ForEach-Object { $_.Name })
```

Doesn't make a behavioral difference with `-notin` but I figure why not be safe.